site/book/04-using-functions/02-imperative-function-execution.md

```diff
@@ -5,7 +5,7 @@ For example, to set the namespace of all resources in the wordpress package
 hierarchy:
 
 ```shell
-$ kpt fn eval wordpress --image gcr.io/kpt-fn/set-namespace:v0.1 -- namespace=mywordpress
+$ kpt fn eval wordpress -i set-namespace:v0.1 -- namespace=mywordpress
```

Comment:
Can you you show the long form first:

```$ kpt fn eval wordpress --image gcr.io/kpt-fn/set-namespace:v0.1 -- namespace=mywordpress```

and then say explicitly there's a short-hand form for using functions with this prefix. Otherwise, just showing the short-hand makes it seem like you can't have custom functions outside this registry. Same in declarative flow.

Reply:
Added.